README.md

```diff
@@ -106,7 +106,7 @@ k8s_defaults(
   name = "k8s_deploy",
   kind = "deployment",
   # This is the name of the cluster as it appears in:
-  #   kubectl config current-context
+  #   kubectl config view --minify -o=jsonpath='{.contexts[0].context.cluster}'
```

Comment:
@nlopezgi 
Context is the combination of user + cluster. By default gcloud will set user, cluster and context to be the same thing (probably why this bug exists) but that need not be the case. Totally possible that other distros besides GKE do something different.

Context may actually be what most people want here (and is typically what people use). Probably worth providing a `context` parameter here in addition to `cluster` (which is probably better than repurposing cluster to be cluster).

@mattmoor can you comment on the history here? Is this an accident or intentional?